src/actions/channels.js

```diff
@@ -690,6 +690,9 @@ export function viewChannel(channelId: string, prevChannelId: string = ''): Acti
         const {myPreferences} = getState().entities.preferences;
         const viewTimePref = myPreferences[`${Preferences.CATEGORY_CHANNEL_APPROXIMATE_VIEW_TIME}--${channelId}`];
         const viewTime = viewTimePref ? parseInt(viewTimePref.value, 10) : 0;
+        const pchanMannuallyUnread = isManuallyUnread(prevChannelId, getState());
```

Comment:
Minor nit, I'd avoid mixing naming styles for the previous channel ID to make this a bit easier to follow
```suggestion
        const prevChanManuallyUnread = isManuallyUnread(prevChannelId, getState());
```